plugins/modules/fabric_virtual_networks_workflow_manager.py

```diff
@@ -0,0 +1,2950 @@
+                ).format(verify_anycast_list)
+                self.log(msg, "INFO")
+            else:
+                msg = (
+                    "Playbook's input does not match with Cisco Catalyst Center, indicating that Anycast "
+                    " Gateway(s) '{0}' deletion task may not have executed successfully."
+                ).format(missed_anycast_list)
+                self.log(msg, "INFO")
+
+        return self
+
+
+def main():
+    """ main entry point for module execution
+    """
+
+    element_spec = {'dnac_host': {'required': True, 'type': 'str'},
+                    'dnac_port': {'type': 'str', 'default': '443'},
+                    'dnac_username': {'type': 'str', 'default': 'admin', 'aliases': ['user']},
+                    'dnac_password': {'type': 'str', 'no_log': True},
+                    'dnac_verify': {'type': 'bool', 'default': 'True'},
+                    'dnac_version': {'type': 'str', 'default': '2.2.3.3'},
+                    'dnac_debug': {'type': 'bool', 'default': False},
+                    'dnac_log_level': {'type': 'str', 'default': 'WARNING'},
+                    "dnac_log_file_path": {"type": 'str', "default": 'dnac.log'},
+                    "dnac_log_append": {"type": 'bool', "default": True},
+                    'dnac_log': {'type': 'bool', 'default': False},
+                    'validate_response_schema': {'type': 'bool', 'default': True},
+                    'config_verify': {'type': 'bool', "default": False},
+                    'dnac_api_task_timeout': {'type': 'int', "default": 1200},
+                    'dnac_task_poll_interval': {'type': 'int', "default": 2},
+                    'config': {'required': True, 'type': 'list', 'elements': 'dict'},
+                    'state': {'default': 'merged', 'choices': ['merged', 'deleted']}
+                    }
+
+    module = AnsibleModule(argument_spec=element_spec,
+                           supports_check_mode=False)
+
+    ccc_virtual_network = VirtualNetwork(module)
+    state = ccc_virtual_network.params.get("state")
+
+    if state not in ccc_virtual_network.supported_states:
+        ccc_virtual_network.status = "invalid"
+        ccc_virtual_network.msg = "State {0} is invalid".format(state)
+        ccc_virtual_network.check_return_status()
+
+    ccc_virtual_network.validate_input().check_return_status()
+    config_verify = ccc_virtual_network.params.get("config_verify")
+
+    for config in ccc_virtual_network.validated_config:
+        ccc_virtual_network.reset_values()
+        ccc_virtual_network.get_want(config).check_return_status()
+        ccc_virtual_network.get_have(config).check_return_status()
+        ccc_virtual_network.get_diff_state_apply[state](config).check_return_status()
+        if config_verify:
+            ccc_virtual_network.verify_diff_state_apply[state](config).check_return_status()
+
+    # Invoke the API to check the status and log the output of each fabric vlan(s), virtual network(s)
```

Comment:
# Invoke the API to check status and log the output of each fabric VLAN, virtual network, and anycast gateways update